app/code/community/Ebizmarts/MailChimp/Helper/Data.php

```diff
@@ -1101,19 +1100,10 @@ public function getWebhooksKey()
      */
     public function resetErrors($scopeId, $scope = 'stores')
     {
-        // reset subscribers with errors
-        $collection = Mage::getResourceModel('newsletter/subscriber_collection')
-            ->addFieldToFilter('mailchimp_sync_error', array('neq' => ''));
+        $where = "mailchimp_sync_error <> ''";
```

Comment:
$where variable should be created inside the removeMessageErrorsFromNewsletterSubscriber function.

Reply:
Solved